### PR TITLE
registry: Change default endpoint on windows to a windows-specific one

### PR DESCRIPTION
**Pull request by tiborvass**
Signed-off-by: Tibor Vass tibor@docker.com

Ping @icecrime @dmp42

### CODE REVIEW

**Comment by tiborvass** [comment]
**This disables entirely the v1 registry on windows. Let me know if that's not desired.**

**Comment by icecrime** [comment]
LGTM :+1:

**Comment by dmp42** [comment]
LGTM

File: registry/consts_windows.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// +build windows
+
+package registry
```
Comment (stevvooe) [inline registry/consts_windows.go]: This needs a big comment explaining why this was done and that it is temporary.

Comment (tiborvass) [inline registry/consts_windows.go]: ok

**Comment by calavera** [comment]
is `https://ms-tp3.registry-1.docker.io` up and working? I'm guessing all tests on windows that depend on the registry will fail if that host is not ready.

**Comment by tiborvass** [comment]
@calavera the CI doesn't test Windows daemon yet, so that codepath is never executed on the CI.

**Comment by calavera** [comment]
okay, it makes sense. Carry on! :+1:

**Comment by thaJeztah** [comment]
@tiborvass I added some impact labels, but please review them, if I added the correct ones

**Comment by tiborvass** [comment]
@thaJeztah I don't think this should have any impact at all. This is solely for the windows daemon which is not released yet.

**Comment by thaJeztah** [comment]
@tiborvass doh! I read "blocking v1 registry", didn't really think "daemon".

yeah, I doubt many will be affected :smile:

**Pull request merged**